ioctl/doc/doc.go

```diff
@@ -32,19 +32,17 @@ func GenMarkdownTreeCustom(c *cobra.Command, dir string, name string, path strin
 		filename = filepath.Join(path, "README.md")
 	}
 
-	f, err := os.Create(filename)
+	f, err := os.Create(filepath.Clean(filename))
 	if err != nil {
 		return err
 	}
-	defer f.Close()
```

Comment:
can you check why security scan considers `f.Close()` as "unsafe method"? does it mean the error is not checked? like you need to:
```
defer func() {
    return f.Close()
}
```

Reply:
error is skipped by `defer f.Close()`. it's unsafe.  error should return. 
`defer func() error {
    return f.Close()
}` 
 or  `return f.Close()` at the end of processing is both ok

Reply:
i see, so should check/catch the error inside `defer`

if `return f.Close()` at the end, also need to handle `f.Close()` in L42 and L45, so defer is better

Reply:
current code is acceptable, you could find a new way to improve it

Reply:
ok